Assets/Scripts/Environment/Hangar/Battery/BatteryInteractable.cs

```diff
@@ -10,6 +12,11 @@ public class BatteryInteractable : InteractableObject
     // Shows that the player is holding any battery
     public static bool PlayerIsHoldingABattery;
 
+    [SerializeField] private UnityEvent takeBatteryEvent;
```

Comment:
For the Player animations, I don't think we necessarily need events. We should able to call the animator functions directly. Events can be a bit messy/expensive/hard to debug too.

The ```PlayerManager``` could expose the animator. 

Maybe we need a new component for handling player animations/animations of this kind, if we do it like this. 

Events could be nice for handling other kinds of animations though. I just don't think it makes sense for the Player, but if there's a good reason we can roll with it.